package.json

```diff
@@ -10,87 +10,27 @@
 		"type": "git",
 		"url": "git+https://github.com/guardian/commercial-core.git"
 	},
-	"license": "Apache-2.0",
-	"main": "dist/cjs/index.js",
-	"module": "dist/esm/index.js",
-	"files": [
-		"dist"
-	],
 	"scripts": {
-		"build": "npm-run-all clean --parallel compile:*",
-		"clean": "rm -rf dist",
-		"commit": "cz",
-		"compile:common": "tsc --project ./tsconfig.build.json --outDir ./dist/cjs --module CommonJS",
-		"compile:esm": "tsc --project ./tsconfig.build.json --outDir ./dist/esm",
-		"lint": "eslint .",
-		"prettier:check": "prettier . --check --cache",
-		"prettier:fix": "prettier . --write --cache",
 		"precommit:lint": "lint-staged",
-		"prepush:test": "jest --verbose --runInBand --onlyChanged",
-		"test": "jest",
-		"test:watch": "jest --watch",
-		"test:ci": "jest --coverage --ci",
-		"tsc": "tsc --noEmit",
-		"validate": "npm-run-all tsc lint test build",
-		"prepare": "husky install"
+		"prepush:test": "yarn workspace @guardian/commercial-core jest --verbose --runInBand --onlyChanged"
 	},
+	"workspaces": [
+		"core",
+		"standalone"
```

Comment:
I'm not super keen on the name 'standalone' as it originates from when we used to have separate bundles for Frontend and DCR. But we can think of better names later.

Reply:
Agreed!